### PR TITLE
Master himan

### PR DESCRIPTION
**Pull request by himanshu81494**
updated: 
1 typo in readme file
2 use of goto in C
3 call by reference example
4 added some text in json file

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@ commented code and explained as they go.
 
 ... to write more inline code tutorials. Just grab an existing file from
 this repo and copy the formatting (don't worry, it's all very simple).
-Make a new file, send a pull request, and if it passes muster I'll get it up pronto.
+Make a new file, send a pull request, and if it passes master I'll get it up pronto.
```
Comment (ian-bertolacci) [inline README.markdown]: "muster" is the correct word here.
Here it means to pass inspection.

Comment (himanshu81494) [inline README.markdown]: ok got it and corrected it.

**Comment by levibostian** [comment]
Sorry, can you merge in master?

**Comment by vendethiel** [comment]
@himanshu81494 pretty please? :)

**Comment by himanshu81494** [comment]
well, I am sorry @vendethiel if you were offended !

**Pull request closed without merging**